stdlib/public/TensorFlow/Ops.swift

```diff
@@ -1600,25 +1600,36 @@ public extension Tensor {
 
 public extension Tensor {
   @inlinable
+  // SWIFT_ENABLE_TENSORFLOW
+  @differentiable(wrt: self, vjp: _vjpBroadcast(toShape:)
```

Comment:
Only this function needs a custom derivative. Note that all other functions `braodcast(to:)` and `broadacast(like:)` are defined in terms of `broadcast(toShape:)`. So for `braodcast(to:)` and `broadacast(like:)` you only need to mark them `@differentiable` but not provide a VJP.

Reply:
`func broadcast(to:)` uses `map` in the function body, and from what I recall, we don't currently support this (until control flow will be finished). So I see that I don't need a `broadcast(like:)` function as it just uses `broadcast(toShape:)`. So it seems I need a total of 4 JVP functions instead of 6.